docs/models_repository.md

```diff
@@ -20,19 +23,22 @@ models/
 │       ├── ir_model.bin
 │       └── ir_model.xml
 └── model2
+│   └── 1
+│   │   ├── ir_model.bin
+│   │   ├── ir_model.xml
+│   │   └── mapping_config.json
+└── model3
     └── 1
-        ├── ir_model.bin
-        ├── ir_model.xml
-        └── mapping_config.json
+        └── model.onnx
 ``` 
 
 - Each model should be stored in a dedicated directory (model1 and model2 in the examples above) and should include sub-folders
 representing its versions (1,2, etc). The versions and the sub-folder names should be positive integer values. 
 
-- Every version folder _must_ include a pair of model files with .bin and .xml extensions; however, the file name can be arbitrary.
+- Every version folder _must_ include model files. IR model files must be with .bin and .xml extensions. ONNX model must have .onnx extension. The file name can be arbitrary.
 
-- Each model in IR format defines input and output tensors in the AI graph. By default OpenVINO&trade; model server is using 
-tensors names as the input and output dictionary keys.  The client passes the input values to the gRPC request and 
+- Each model defines input and output tensors in the AI graph. By default OpenVINO&trade; model server is using 
+tensors names as the prediction input and output names.  The client passes the input values to the gRPC request and 
```

Comment:
"By default OpenVINO&trade; model server is using tensors names as the prediction input and output names"

It's a little bit  confusing at the first sight. Maybe we could reformulate it to be more descriptive?
Also we could drop gRPC here as the same occurs for HTTP requests.

Take a look at below proposal:
"Each model defines input and output tensors in the AI graph. 
The client passes data to model input tensors by filling appropriate entries in the request inputs map.
Prediction results appear in model output tensors when inference is finished and can be read from response outputs map. By default OpenVINO&trade; model server is using model
tensors names as inputs and outputs names in requests and responses. "

Does it make sense?